src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -217,6 +217,18 @@ public static String formatDate(Date date) {
         return sdf.format(date);
     }
 
+    /**
+     * Formats a date in the format D, dd MMM, yyyy. (Example: Sat, 05 May, 2012)
```

Comment:
Remove the brackets for consistency with the other comments? (If you were trying to restart the build, one way is to close and reopen the PR).

Reply:
@whipermr5 Yes, that's what I was trying to do. Next time I'll close and reopen the PR. Thanks.